.github/workflows/image-update.yaml

```diff
@@ -7,8 +7,8 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
       - uses: actions/checkout@v1
-      - run: make -f Makefile.docker eksctl-image
```

Comment:
The idea here is to build the image, then update manifests and push new image. We are currently not able to actually push the image, it's something that we need to do.